### ISSUE TITLE
Failed Login page parsing

### ISSUE THREAD

**Issue by ldoktor** [cgeo/cgeo#6503]
##### Detailed steps causing the problem:
- Start cgeo
- Toast says "Unable to login to ..."
- Go to settings->services->geocaching.com->authenticate
- Fill in my credentials and press authenticate

##### Actual behavior after performing these steps:
Pop-up says `Failed Login page parsing` (`Přihlášení se nezdařilo: Čtení přihlašovací obrazovky selhalo`)

##### Expected behavior after performing these steps:
I should be logged-in (credentials are valid, the page loads well when using the "Open web page" button bellow)

##### Version of c:geo used:
2016.07.15-NB-119f75b

##### Is the problem reproducible for you?
Yes, always

##### System information:
```
--- System information ---
Device: 2014811 (cm_wt88047, Xiaomi)
Android version: 6.0.1
Android build: cm_wt88047-userdebug 6.0.1 MOB30Z 72b64f0fbc test-keys
Cgeo version: 2016.07.15-NB-119f75b
Google Play services: enabled
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: false
Map strategy: auto
HW acceleration: enabled (default state)
System language: cs_CZ
Log date format: 19.04.17
Debug mode active: no
Geocaching sites enabled:
geocaching.com: Not logged in (Nejsou uloženy žádné přihlašovací údaje)
Installed cgeo plugins: none
--- End of system information ---
```

##### Other comments and remarks:
* This occurred while starting this app after ~ a year (or so), before it always worked well (thank you for this amazing app)
* My cellphone is in Czech language, I used the `strings.xml` to translate the login failure. For the steps to reproduce the problem I translated it myself so it might not be accurate.
* I also resetting the cgeo data from android and set everything from scratch with the same results.

**Comment by kumy** [comment]
> Cgeo version: 2016.07.15-NB-119f75b

Your version seems quite outdated. GroundSpeak has made some changes to the login page recently. You should update.

**Comment by Lineflyer** [comment]
Yes, that is the reason.
You are using a one year old alpha version.

Please kindly either load the current official version 2017.03.29 from Google Play or Github, or use an up to date nightly version.

**Comment by ldoktor** [comment]
I'm really sorry about reporting this issue. I triple checked google-play updates but it was happy so I thought there were no updates, but it turned out to just incorrectly report the version because the year ago I used a custom build (because of another issue).

Pure uninstall and install did not work, I had to use `adb uninstall cgeo.geocaching` and then the `adb install ./cgeo-release_20170329.apk` worked well.

Thank you for the prompt answers.
